core/src/wallet/common/synchronizers/AbstractBlockchainExplorerAccountSynchronizer.h

```diff
@@ -269,10 +269,19 @@ namespace ledger {
                         //Check if tx is pending
                         auto it = buddy->savedState.getValue().pendingTxsHash.find(tx.first);
                         if (it == buddy->savedState.getValue().pendingTxsHash.end()) {
-                            buddy->logger->info("Drop transaction {}", tx.first);
-                            buddy->logger->info("Deleting operation from DB {}", tx.second);
-                            //delete tx.second from DB (from operations)
-                            sql << "DELETE FROM operations WHERE uid = :uid", soci::use(tx.second);
+                            {
```

Comment:
That scope is not necessary.

Reply:
Good catch, remains test + logging phase 😅